elements/Image.js

```diff
@@ -19,7 +19,7 @@ export default class extends Shape {
         y: numberProp,
         width: numberProp.isRequired,
         height: numberProp.isRequired,
-        href: Image.propTypes.source,
+        href: PropTypes.object,
```

Comment:
it's not very accurate, source can also be a `number` when image is `require`d
https://github.com/facebook/react-native/blob/master/Libraries/Image/ImageSourcePropType.js#L82-L88

const ImageSourcePropType = PropTypes.oneOfType([
  ImageURISourcePropType,
  // Opaque type returned by require('./image.jpg')
  PropTypes.number,
  // Multiple sources
  PropTypes.arrayOf(ImageURISourcePropType),
]);

Reply:
will update accordingly.